Sources/Network/Pinning/ServerTrustEvaluator+Configuration.swift

```diff
@@ -96,13 +96,9 @@ extension ServerTrustEvaluator {
     }
 }
 
-extension ServerTrustEvaluator.PinningPolicy: Hashable {
+extension ServerTrustEvaluator.PinningPolicy: Equatable {
 
     public static func == (lhs: ServerTrustEvaluator.PinningPolicy, rhs: ServerTrustEvaluator.PinningPolicy) -> Bool {
         return lhs.domainName == rhs.domainName
     }
-
-    public var hashValue: Int {
```

Comment:
I think we also need this custom implementation so that only the domain is considered for the hash, which is the desired behaviour (uniquing factor) when adding policies to a `Set`